python/lsst/afw/table/aggregates.cc

```diff
@@ -156,17 +160,24 @@ PYBIND11_PLUGIN(_aggregates) {
     clsEllipseKey.def(py::init<SubSchema const &>());
 
     /* Operators */
+    // Unlike the other keys, CoordKey uses a free function to test equality
```

Comment:
@TallJimbo may know why CoordKey defines `operator==` differently than all other keys in `aggregates.h` (defining it as a free function instead of an operator method). I agree it would be nice to change this if it can be changed. But do we want to do that now, or wait until after the pybind11 merge? If the latter I can open a ticket (though it is likely to get lost and forgotten).